src/Opg/Lpa/Pdf/Service/Forms/Lp3.php

```diff
@@ -23,45 +23,26 @@ public function __construct(Lpa $lpa)
         $this->generatedPdfFilePath = $this->getTmpFilePath('PDF-LP3');
 
         $this->Lp3Template = $this->pdfTemplatePath."/LP3.pdf";
-        $this->Lp3PageOneTemplate = $this->pdfTemplatePath."/LP3-1.pdf";
-        $this->Lp3PageTwoTemplate = $this->pdfTemplatePath."/LP3-2.pdf";
-        $this->Lp3PageThreeTemplate = $this->pdfTemplatePath."/LP3-3.pdf";
-        $this->Lp3PageFourTemplate = $this->pdfTemplatePath."/LP3-4.pdf";
     }
 
     /**
      * Populate LPA data into PDF forms, generate pdf file and save into file path.
-     *
-     * @return Form object | null
      */
     public function generate()
     {
         $this->logGenerationStatement();
 
         // will not generate pdf if there's no people to notify
         $noOfPeopleToNotify = count($this->lpa->document->peopleToNotify);
-        if($noOfPeopleToNotify == 0) {
+        if ($noOfPeopleToNotify == 0) {
             throw new \RuntimeException("LP3 is not available for this LPA.");
         }
 
-        if($noOfPeopleToNotify == 1) {
-            $this->generateSingleNotificationPdf();
-        }
-        else {
-
-            // Loop over each person, adding a page for them.
-            foreach($this->lpa->document->peopleToNotify as $peopleToNotify) {
-
-                if( $peopleToNotify instanceof NotifiedPerson ){
-                    // Generate a standard notification letter for each person to be notified.
-                    $this->generatePageOnePdf( $peopleToNotify );
-                }
-
+        foreach ($this->lpa->document->peopleToNotify as $peopleToNotify) {

```

Comment:
Can we change the last parameter to $personToNotify here? Just to be clear it involves one entity.